cli/cmd/generate.go

```diff
@@ -63,6 +63,7 @@ subcommands.`,
 	cmd.Flags().StringP("policy", "p", rulesFilename, "path to policy (.rego) file")
 	cmd.Flags().StringP("settings", "s", settingsFilename, "path to settings (.json) file")
 	cmd.Flags().StringP("manifest", "m", manifestFilename, "path to manifest (.json) file")
+	cmd.Flags().String("reference-values", "", "set the default reference values used for attestation (one of: aks)")
```

Comment:
`(one of: aks)` reads a bit strange. We could print a detailed list of embedded defaults with `--version`.

Reply:
@davidweisse please create a follow up ticket to explicitly print the relevant values in a separate section as AKS defaults.